README.md

```diff
@@ -5,8 +5,38 @@
 
 # Thorgate's collection of utility packages for Single-page applications
 
-This is combination of utility packages used at Thorgate
+This is combination of utility packages used at [Thorgate](https://thorgate.eu).
 
+## Packages included:
+
+Some of the packages are used by [Thorgate project template (SPA variant)](https://gitlab.com/thorgate-public/django-project-template/tree/spa).
+
+* [components](packages/components/README.md) - common components used with SPA template
+* [create-resource-saga](packages/create-resource-saga/README.md) -  helper for creating [Redux-Saga](https://github.com/redux-saga/redux-saga)s loading resources
+* [entities](packages/entities/README.md) - normalized data storage helper
+* [errors](packages/errors/README.md) - error handling helpers used by SPA template
+* [forms](packages/forms/README.md) - helper for creating [Redux-Saga](https://github.com/redux-saga/redux-saga)s for saving forms
+* [is](packages/is/README.md) - internal type checking library for packages in SPA template
+* [loading-bar](packages/loading-bar/README.md) - controlled loading bar component
+* [named-routes](packages/named-routes/README.md) - routes helper library to add named routes via [react-router-config](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-config)
+* [pending-data](packages/pending-data/README.md) - pending data manager used by SPA template
+* [permissions](packages/permissions/README.md) - permissions helpers used by SPA template
+* [saga-manager](packages/saga-manager/README.md) - helper to hot-reload [Redux-Saga](https://github.com/redux-saga/redux-saga)s
+* [view](packages/view/README.md) - [Redux-Saga](https://github.com/redux-saga/redux-saga) runners used by SPA template
+* [view-manager](packages/view-manager/README.md) - view helpers used by SPA template 
+
+
+## Running the example app
+
+To run the example web application, install the dependencies and build the packages first:
+
+```
+yarn install
```

Comment:
Initial commands should be
```
yarn install
yarn bootstrap
```

Contribution guide has some of the info as well, but main readme would be found more easily

Reply:
Added a link to CONTRIBUTING.md, fixed bootstrap commands list.